### PR TITLE
create different types of components

### PR DESCRIPTION
**Pull request by Arzamas-ka**
#### Create components in different ways, using:  

1. React.CreateElement
2. React.Component
3. React.PureComponent
4. Functional component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,32 @@
-import logo from './logo.svg';
```
Comment (osonovaakylai) [inline src/App.js:1]: Please use **jsx** extension in all React component files

File: src/components/FunctionalComponent.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const FunctionalComponent = () => {
+  return (
```
Comment (osonovaakylai) [inline src/components/FunctionalComponent.js:2]: In function components the word "return" can be omitted
Suggested change
```suggestion
import React from "react";

const FunctionalComponent = () => (
  <>
    <h1>Hello World!</h1>
    <span>It's a Functional.Component</span>
    <hr />
  </>
);

export default FunctionalComponent;
```

**Comment by Arzamas-ka** [comment]
@osonovaakylai comments fixed

**Review by osonovaakylai — APPROVED** [review]
There are some remarks. It would be better to discuss them in the meeting

**Comment by Arzamas-ka** [comment]
> There are some remarks. It would be better to discuss them in the meeting

ok

**Review by osonovaakylai — APPROVED** [review]

**Pull request merged**